core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/login/LoginModifierConfig.java

```diff
@@ -0,0 +1,148 @@
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import java.util.Map;
+
+public class LoginModifierConfig {
+
+  private static final String DEFAULT_LOGIN = "admin";
+
+  private static final String DEFAULT_PASSWORD = "admin";
+
+  private static final String DEFAULT_LOGIN_INPUT_SELECTOR = "//input[@name='j_username']";
+
+  private static final String DEFAULT_PASSWORD_INPUT_SELECTOR = "//input[@name='j_password']";
+
+  private static final String DEFAULT_SUBMIT_BUTTON_SELECTOR = "//*[@type='submit']";
+
+  private static final String DEFAULT_LOGIN_TOKEN = "login-token";
+
+  private static final String LOGIN_PARAM = "login";
+
+  private static final String PASSWORD_PARAM = "password";
+
+  private static final String LOGIN_PAGE_PARAM = "login-page";
+
+  private static final String LOGIN_INPUT_SELECTOR_PARAM = "login-input-selector";
+
+  private static final String PASSWORD_INPUT_SELECTOR_PARAM = "password-input-selector";
+
+  private static final String SUBMIT_BUTTON_SELECTOR_PARAM = "submit-button-selector";
+
+  private static final String LOGIN_TOKEN_KEY_PARAM = "login-token-key";
+
+  private static final String LOGIN_CHECK_TIMEOUT_PARAM = "timeout";
+
+  private static final String FORCE_LOGIN = "force-login";
+
+  private static final String DEFAULT_CHECK_TIMEOUT_PARAM = "1000";
+
+  private static final String RETRIAL_NUMBER_PARAM = "retrial-number";
+
+  private static final String DEFAULT_RETRIAL_NUMBER = "3";
+
+  private final String loginPage;
+
+  private final String login;
+
+  private final String password;
+
+  private final String loginInputSelector;
+
+  private final String passwordInputSelector;
+
+  private final String submitButtonSelector;
+
+  private final String loginTokenKey;
+
+  private final int loginCheckTimeout;
+
+  private final boolean forceLogin;
+
+  private final int retrialNumber;
+
+  public LoginModifierConfig(Map<String, String> params) throws ParametersException {
+    this.loginPage = params.get(LOGIN_PAGE_PARAM);
+    this.login = getParameter(params, LOGIN_PARAM, DEFAULT_LOGIN);
+    this.password = getParameter(params, PASSWORD_PARAM, DEFAULT_PASSWORD);
+    this.loginInputSelector = getParameter(params, LOGIN_INPUT_SELECTOR_PARAM, DEFAULT_LOGIN_INPUT_SELECTOR);
+    this.passwordInputSelector = getParameter(params, PASSWORD_INPUT_SELECTOR_PARAM, DEFAULT_PASSWORD_INPUT_SELECTOR);
+    this.submitButtonSelector = getParameter(params, SUBMIT_BUTTON_SELECTOR_PARAM, DEFAULT_SUBMIT_BUTTON_SELECTOR);
+    this.loginTokenKey = getParameter(params, LOGIN_TOKEN_KEY_PARAM, DEFAULT_LOGIN_TOKEN);
+    this.forceLogin = BooleanUtils.toBoolean(params.get(FORCE_LOGIN));
+    int loginTimeout = NumberUtils.toInt(getParameter(params, LOGIN_CHECK_TIMEOUT_PARAM, DEFAULT_CHECK_TIMEOUT_PARAM));
+    this.loginCheckTimeout = Math.min(10000, loginTimeout);
```

Comment:
Could we rename this attribute to sth like:
- `this.waitForAuthentication = loginCheckDelay`
- `this.delayBeforeLoginCheck = delay`
- ??